### ISSUE TITLE
Class labels and dataset download

### ISSUE THREAD

**Issue by kevinjose** [nasa/pretrained-microscopy-models#11]
Hi,
I have two questions:
1) Is the MicroNet dataset publically available? I could not find a link to it in the paper/code/from googling
2) What are the class labels in the classfication task? I could not find them in the paper/code. The example simply outputs an index.

Thanks!
Kevin

**Comment by JStuckner** [comment]
Hi Kevin!

1. Unfortunately, MicroNet isn't publicly available. I really wish I could have released it, but some of the data is sensitive. About 1/3 of the data in the dataset is publicly sourced. The community would greatly benefit from a large publicly available set of labelled micrographs.
2. Index 0 is Al 6061, the next 10 are the Aversa classes. There are high temp superalloys, ceramics, powders, and alloys. The main purpose of the encoders is to do transfer learning. So you can take off the last classification layer and train the models on your own classes. Or you can plug the encoder into a segmentation or regression model and get better results with less training data. We intend to release more examples of other tasks later this summer.

Sorry I can't be more helpful.
Josh

**Comment by kevinjose** [comment]
Understood! Many thanks!

**Comment by GuoChi-Li** [comment]
Hi JStuckner,

As you mentioned, MicroNet isn't publicly available because of the sensitive data.
Is it possible to share the non-sensitive data with the public?

Thanks.
Guo-Chi

**Comment by JStuckner** [comment]
Hi Guo-Chi,

Hmm, it might be possible for me to share some data. However, it would be a large effort to get it approved by our various scientists, lawyers, and export control. It would likely take months. I would be willing to do it if I had partners that were also willing to release significant amounts of data that we could put together into a single large dataset.

Regards,
Josh

**Comment by GuoChi-Li** [comment]
Hi JStuckner,

It would be great if you could share the data. It will definitely benefit the community. Unfortunately, I don't have a large dataset. 

Regards,
Guo-Chi
